components/mpas-ocean/src/mode_init/mpas_ocn_init_global_ocean.F

```diff
@@ -1183,9 +1187,15 @@ subroutine ocn_init_setup_global_ocean_interpolate_land_ice_topography(domain, i
           call mpas_pool_get_array(landIceInitPool, 'landIceGroundedFracObserved', landIceGroundedFracObserved)
           call mpas_pool_get_array(statePool, 'ssh', ssh, 1)
           call mpas_pool_get_array(forcingPool, 'landIceMask', landIceMask)
+          call mpas_pool_get_array(forcingPool, 'landIceFloatingMask', landIceFloatingMask)
           call mpas_pool_get_array(forcingPool, 'landIcePressure', landIcePressure)
           call mpas_pool_get_array(forcingPool, 'landIceFraction', landIceFraction)
+          call mpas_pool_get_array(forcingPool, 'landIceFloatingFraction', landIceFloatingFraction)
 
+          if (landIceFloatingMask(1) == -1) then
+             call mpas_log_write('landIceFloatingMask contains the default value which is invalid.', &
+                                 MPAS_LOG_CRIT)
+          endif
```

Comment:
This is great!  However, if I were a user and got this error, I wouldn't understand what had gone wrong.  Can we change the message to:
```suggestion
          if (landIceFloatingMask(1) == -1) then
             call mpas_log_write('landIceFloatingMask contains the default value which likely indicates that this field is missing in the initial condition file (e.g. because it is meant for an older E3SM version).', &
                                 MPAS_LOG_CRIT)
          endif
```